### PR TITLE
Expose "created date" to views

### PR DESCRIPTION
**Pull request by laryn**
This allows date formatting options on the `created date` in views.

### CODE REVIEW

File: modules/views/components/civicrm.core.inc [hunk 1]
```diff
@@ -721,6 +721,22 @@ function _civicrm_core_data(&$data, $enabled) {
...
+  $data['civicrm_contact']['created_date'] = array(
+    'title' => t('Created date'),
+    'help' => t("The date the contact's record was created."),
+    'field' => array(
+      'handler' => 'civicrm_handler_field_datetime',
+      'click sortable' => TRUE,
+    ),
+    'filter' => array(
+      'handler' => 'civicrm_handler_filter_datetime',
+      'is date' => TRUE,
```
Comment (seamuslee001) [inline modules/views/components/civicrm.core.inc:734]: @laryn just a thought should we allow for NULL and NOT NULL filters here as well?

Comment (laryn) [reply modules/views/components/civicrm.core.inc:734]: @seamuslee001 I'm definitely open to that. I simply copied the code from the `modified_date` field in the same area:
https://github.com/civicrm/civicrm-drupal/blob/7.x-master/modules/views/components/civicrm.core.inc#L708-L723

Should NULL and NOT NULL be added to that one as well?

Comment (laryn) [reply modules/views/components/civicrm.core.inc:734]: Further thought: Is there ever a case where a created or modified date would be NULL?

**Comment by seamuslee001** [comment]
Jenkins retest this please

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by seamuslee001** [comment]
Jenkins re test this please

File: modules/views/components/civicrm.core.inc (hunk 1 above)
Comment (yashodha) [reply modules/views/components/civicrm.core.inc:734]: @laryn no, it shouldn't be the case, the created /modified date would have value always. Modified date = created date if not modified/updated.

**Comment by yashodha** [comment]
@laryn @seamuslee001 looks straight forward, this can be merged as well. Thanks!

**Comment by yashodha** [comment]
@laryn merging this PR

**Pull request merged**